Grbl_Esp32/grbl_trinamic.cpp

```diff
@@ -200,26 +313,57 @@ void trinamic_change_settings() {
     TRINAMIC_X.microsteps(settings.microsteps[X_AXIS]);
     TRINAMIC_X.rms_current(settings.current[X_AXIS] * 1000.0, settings.hold_current[X_AXIS] / 100.0);
 #endif
+#ifdef X2_TRINAMIC
```

Comment:
This too could be a subroutine.  The argument for it is less compelling than above, but still it would be worthwhile for maintenance reasons.